modules/core/src/main/kotlin/datamaintain/core/step/check/rules/implementations/ExecutedScriptsNotRemovedCheck.kt

```diff
@@ -0,0 +1,36 @@
+package datamaintain.core.step.check.rules.implementations
+
+import datamaintain.core.script.ExecutedScript
+import datamaintain.core.script.ScriptWithContent
+import datamaintain.core.step.check.rules.ScriptType
+import datamaintain.core.step.check.rules.contracts.FullContextCheckRule
+import java.lang.IllegalStateException
+
+class ExecutedScriptsNotRemovedCheck(
```

Comment:
I would have named that class otherwise: ```ExecutedScriptsStillPresentCheck```. It may be worth of some discussion because this precise name needs to be clear for anyone since it will be the way users will declare that rule. 
@fbreuneval I think your opinion could be useful on that point

Reply:
Proposition de nom : ```SameScriptsAsExecutedCheck```